mlir/utils/performance/ck-benchmark-driver/CMakeLists.txt

```diff
@@ -1,16 +1,37 @@
 find_package(composable_kernel 1.1.0 COMPONENTS device_gemm_operations CONFIG)
 
 if (composable_kernel_FOUND)
+  message(STATUS "Found Composable Kernel Library")
   find_package(hip REQUIRED PATHS /opt/rocm)
   set(LIBS composable_kernel::device_gemm_operations hip::device benchmark-driver-utils)
 
+  add_library(ck-common INTERFACE IMPORTED)
+  target_link_libraries(ck-common INTERFACE ${LIBS})
+  target_compile_options(ck-common INTERFACE "-std=c++17")
+
   add_executable(ck-benchmark-driver EXCLUDE_FROM_ALL ck-benchmark-driver.cpp)
-  target_link_libraries(ck-benchmark-driver PRIVATE ${LIBS})
-  target_compile_options(ck-benchmark-driver PRIVATE "-std=c++17")
+  target_link_libraries(ck-benchmark-driver PRIVATE ck-common)
   set_target_properties(ck-benchmark-driver
     PROPERTIES
     RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin"
   )
+
+  include(FetchContent)
+  FetchContent_Declare(
+    cli11
+    GIT_REPOSITORY https://github.com/CLIUtils/CLI11
+    GIT_TAG        v2.2.0
+  )
+  FetchContent_MakeAvailable(cli11)
```

Comment:
I am not against this, but I think this needs to be discussed a bit
a) License: did you double check we can "use" this project (being all Open Source, I think it should be ok, but is it?)
b) The original idea was to have something self-contained, so that running on different machine/environments/OSs should still work and give comparisons. I am happy to add a dependency, but it needs to be discussed a bit more (I think)

Reply:
I added the reference to the CLI11 

https://github.com/ROCm/rocMLIR/blob/d651d4685b00ca19a3c9bd668b4dc78dfe8eb237/mlir/utils/performance/ck-benchmark-driver/README.md?plain=1#L1-L28

I think this should be fine.